### ISSUE TITLE
Fix mode ignores order: flexible

### ISSUE THREAD

**Issue by EthanRutherford** [hudochenkov/stylelint-order#49]
When running stylelint with the autofix option, groups marked "flexible" will be reordered to match the defined order, instead of being correctly left alone (as "flexible" should indicate)

**Comment by hudochenkov** [comment]
It's a known [caveat](https://github.com/hudochenkov/stylelint-order/blob/master/rules/properties-order/README.md#autofixing-caveats). Unfortunately there is nothing I can do. You can disable autofix for properties order if you like.

```js
{
	"rules": {
		"order/properties-order": [
			[
				{
					order: "flexible",
					properties: [
						"color",
						"font-size",
						"font-weight",
					],
				},
			],
			{
				"disableFix": true
			}
		]
	}
}
```

**Comment by hudochenkov** [comment]
In the next version, the behavior will be much better. It might occasionally change order, but if violations were reported only https://github.com/hudochenkov/stylelint-order/issues/51#issuecomment-341161038.

**Comment by hudochenkov** [comment]
Fixed in [`0.8.0`](https://github.com/hudochenkov/stylelint-order/releases/tag/0.8.0). Thank you for reporting!
